### ISSUE TITLE
Discuss resumption vs external PSK usage

### ISSUE THREAD

**Issue by chris-wood** [tlswg/external-psk-design-team#22]
Starter text from @upros!

```
### External PSK versus Resumption PSK Handling

  [[TODO: this section needs TLS stack implementers guidance]]

  On receipt of a `ClientHello` containing  PSK identity, OpenSSL appears to callback to the application first in order to allow the application to check if the PSK is an externally provisioned PSK: https://github.com/openssl/openssl/blob/master/ssl/statem/extensions_srvr.c#L1093. If the application cannot find the external PSK idenitty in its external cache, the TLS stack will then check its resumption cache. We need to document TLS stack vs. application implementers guidance about the sequencing of handling of External vs. Resumption PSKs.
```

**Comment by chris-wood** [comment]
@upros can you please take this?

**Comment by chris-wood** [comment]
Ping @upros!

**Comment by chris-wood** [comment]
Closed by #32.
